tools/regionalgam/flight-curve.R

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env Rscript
 args = commandArgs(trailingOnly=TRUE)
 source(args[1]) #TODO replace by library(regionalGAM) if available as official package from bioconda
-tryCatch({input = read.table(args[2], header=TRUE,sep="	")},finally={input = read.table(args[2], header=TRUE,sep=",")})
+library(data.table)
+# import file
+input = fread(args[2])
+# retransform to data.frame to allow the calculation by the fonctions from the package
```

Comment:
functions?

Reply:
the functions that are in the file dennis-gam-initial-function.R